# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/PBrowser.ipdl
# Commit: 3f7f3285c842
# Full Hash: 3f7f3285c8424dfd0d607e19a66c9cbff6480d0a
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2022-08-26 21:48:35
# Regressor Bug: 1781129
# File Overlap Count: 1
# Description:
#   Bug 1781129 - Part 3: Remove Shmem overload of GetSurfaceData, r=edgar
#   
#   After the previous changes there was only one consumer left of the Shmem
#   version of GetSurfaceData, which could easily be changed to use BigBuffer,
#   removing the need for that overload.
# ==============================================================================

diff -r ac8bea9c2071 -r 3f7f3285c842 dom/ipc/PBrowser.ipdl
--- a/dom/ipc/PBrowser.ipdl	Fri Aug 26 16:08:05 2022 +0000
+++ b/dom/ipc/PBrowser.ipdl	Fri Aug 26 16:08:06 2022 +0000
@@ -636,7 +636,7 @@
     [Nested=inside_sync] sync DispatchTouchEvent(WidgetTouchEvent event);
 
     async InvokeDragSession(IPCDataTransfer[] transfers, uint32_t action,
-                            Shmem? visualData,
+                            BigBuffer? visualData,
                             uint32_t stride, SurfaceFormat format,
                             LayoutDeviceIntRect dragRect,
                             nsIPrincipal principal, nsIContentSecurityPolicy csp,